python_modules/dagster/dagster/_core/definitions/events.py

```diff
@@ -241,8 +241,12 @@ def __init__(
         output_name: Optional[str] = DEFAULT_OUTPUT,
         metadata_entries: Optional[Sequence[Union[MetadataEntry, PartitionMetadataEntry]]] = None,
         metadata: Optional[Mapping[str, RawMetadataValue]] = None,
-        logical_version: Optional[LogicalVersion] = None,
+        data_version: Optional[DataVersion] = None,
+        logical_version: Optional[DataVersion] = None,
```

Comment:
unless we are exporting LogicalVersion, seems like we should just cut it over in one shot.

Reply:
whoops yeah that was leftover from when this was originally intended to merge pre 1.2